tabpfn_client/client.py

```diff
@@ -15,6 +14,15 @@
 SERVER_CONFIG = OmegaConf.load(SERVER_CONFIG_FILE)
 
 
+def get_client_version() -> str:
+    try:
+        return version('tabpfn_client')
+    except PackageNotFoundError:
+        # Package not found, should only happen during development. Simply return a version number that works
+        # for development.
+        return '5.5.5'
```

Comment:
How can I fix this to get the real version number? I can imagine that in the future we might care about trying our code with the right version number.

Is it fixed with a `pip install -e .`? If so please add this to the comment

Reply:
Yes it works, I will add it to the comment.